frontend/src/components/CurrencyList.tsx

```
@@ -0,0 +1,73 @@
+          placeholder="🔍 Search: AUD, dollar, Australia"
+          onChange={(e) => {
+            if (e.target.value.length < 3) {
+              setFilter("");
+            }
+            if (e.target.value.length >= 3) {
+              setFilter(e.target.value);
+            }
+          }}
+        />
+        <ul className="space-y-1">
+          {currencies.map((c) => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)